src/applications/mhv/secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -661,47 +675,84 @@ const ComposeForm = props => {
               )}
 
           <div className="compose-form-div">
-            <CategoryInput
-              category={category}
-              categoryError={categoryError}
-              setCategory={setCategory}
-              setCategoryError={setCategoryError}
-              setUnsavedNavigationError={setUnsavedNavigationError}
-            />
+            {(!mhvSecureMessagingBlockedTriageGroup1p0 ||
+              (!noAssociations && !allTriageGroupsBlocked)) && (
+              <CategoryInput
+                category={category}
+                categoryError={categoryError}
+                setCategory={setCategory}
+                setCategoryError={setCategoryError}
+                setUnsavedNavigationError={setUnsavedNavigationError}
+              />
+            )}
+            {mhvSecureMessagingBlockedTriageGroup1p0 &&
+              (noAssociations || allTriageGroupsBlocked ? (
+                <ViewOnlyDraftSection
+                  title={FormLabels.CATEGORY}
+                  body={`${RadioCategories[(draft?.category)].label}: ${
+                    RadioCategories[(draft?.category)].description
+                  }`}
+                />
+              ) : (
+                <></>
+              ))}
           </div>
           <div className="compose-form-div">
-            <va-text-input
-              label="Subject"
-              required
-              type="text"
-              id="message-subject"
-              name="message-subject"
-              class="message-subject"
-              data-testid="message-subject-field"
-              onInput={subjectHandler}
-              value={subject}
-              error={subjectError}
-              data-dd-privacy="mask"
-              data-dd-action-name="Compose Message Subject Input Field"
-            />
+            {(!mhvSecureMessagingBlockedTriageGroup1p0 ||
+              (!noAssociations && !allTriageGroupsBlocked)) && (
+              <va-text-input
+                label={FormLabels.SUBJECT}
+                required
+                type="text"
+                id="message-subject"
+                name="message-subject"
+                class="message-subject"
+                data-testid="message-subject-field"
+                onInput={subjectHandler}
+                value={subject}
+                error={subjectError}
+                data-dd-privacy="mask"
+                data-dd-action-name="Compose Message Subject Input Field"
+              />
+            )}
+            {mhvSecureMessagingBlockedTriageGroup1p0 &&
+              (noAssociations || allTriageGroupsBlocked ? (
+                <ViewOnlyDraftSection
+                  title={FormLabels.SUBJECT}
+                  body={subject}
+                />
+              ) : (
+                <></>
```

Comment:
same as comment above, see if `&&` is applicable here of `? : ` ternary operator